site_scons/env_modules.py

```diff
@@ -26,7 +26,7 @@
 import distro
 import subprocess
 from subprocess import PIPE, Popen
-from distutils.spawn import find_executable
+from SCons.Script import WhereIs
```

Comment:
(pylint-no-name-in-module) No name 'Script' in module 'SCons'
(pylint-syntax-error) Cannot import 'SCons.Script' due to syntax error 'invalid syntax (<unknown>, line 58)'